WordPress/Classes/ViewRelated/Domains/Domain registration/RegisterDomainSuggestions/RegisterDomainSuggestionsViewController.swift

```diff
@@ -47,6 +41,19 @@ class RegisterDomainSuggestionsViewController: UIViewController {
         return buttonViewController
     }()
 
+    private lazy var transferFooterView = RegisterDomainTransferFooterView()
+
+    /// Represents the layout constraints for the transfer footer view in its visible and hidden states.
+    private lazy var transferFooterViewConstraints: (visible: [NSLayoutConstraint], hidden: [NSLayoutConstraint]) = {
+        let base = [
+            transferFooterView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            transferFooterView.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+        ]
+        let visible = base + [transferFooterView.bottomAnchor.constraint(equalTo: view.bottomAnchor)]
+        let hidden = base + [transferFooterView.topAnchor.constraint(equalTo: view.bottomAnchor)]
+        return (visible: visible, hidden: hidden)
+    }()
```

Comment:
This is an approach that I remember watching in a WWDC session to animate a view that has constraints. In this case, I want to animate the view from "Visible" state to "Hidden" state, and to achieve that:
- I created constraints for the "Visible" state and "Hidden" state.
- I toggle between "Visible" and "Hidden" state simply by activating / deactivating constraints. See this [code](https://github.com/wordpress-mobile/WordPress-iOS/pull/22016/files#diff-5d85c2705cf929cffd73e321a41818e6acbb1f24621f28bd5c4e232995f3b840R159).